src/main/java/com/redhat/rhjmc/containerjfr/platform/ServiceRef.java

```diff
@@ -49,16 +49,14 @@
 
     private final String connectUrl;
     private final String alias;
-    private final int port;
 
-    public ServiceRef(String connectUrl, int port) {
-        this(connectUrl, connectUrl, port);
+    public ServiceRef(String connectUrl) {
+        this(connectUrl, connectUrl);
     }
 
-    public ServiceRef(String connectUrl, String alias, int port) {
+    public ServiceRef(String connectUrl, String alias) {
```

Comment:
The point of the change is to ensure that all `PlatformClients` report JMXServiceURLs, right? So, should the `String connectUrl` here perhaps be replaced by a `JMXServiceURL`?